Server plugins - improve the plugin section and mention a CLI helper tool (#7272)
qgis/QGIS-Documentation: docs/server_manual/plugins.rst

   .. contents::
      :local:

List of plugins
===============

Plugins can also be installed on QGIS Server. They can add new features or alter existing ones from
QGIS Server.

Some plugins designed for server can be found on the official
`QGIS repository <https://plugins.qgis.org/plugins/server/>`_.

Location of plugins
===================

By default, on Debian based systems, QGIS Server will look for plugins located in
:file:`/usr/lib/qgis/plugins`. The default value is displayed when QGIS Server is starting, in the logs.
It's possible to set a custom path by defining the environment variable ``QGIS_PLUGINPATH``

> in the Nginx or Apache configuration.

Reviewer: ```suggestion
in the web server configuration.
```

(to be more generic?)

 unzip master.zip
 mv qgis-helloserver-master HelloServer

With a command line tool
~~~~~~~~~~~~~~~~~~~~~~~~

> If you need to install and upgrade plugins on regularly basis which are stored in a QGIS plugin repository,

Reviewer 2: ```suggestion
If you need to install and regularly upgrade plugins which are stored in the QGIS plugin repository,
```
